.github/workflows/e2e-tests.yml

```diff
@@ -16,10 +18,26 @@ jobs:
     name: e2e-tests
     env:
       PREMIUM_EMBEDDING_TOKEN: ${{ secrets.ENTERPRISE_TOKEN }}
+      METABASE_APP_DB_USER: ${{ secrets.APP_DB_USER }}
+      METABASE_APP_DB: ${{ secrets.APP_DB_SHOPPY_DATABASE }}
+    permissions:
+      id-token: write
```

Comment:
I haven't seen this before, but it should be safe.

> The id-token: write permission allows the GITHUB_TOKEN to make an API call to generate an OIDC token (which is an extra security token, a standard one). The OIDC token is not dangerous in itself. It is minted and signed by GitHub and includes "claims"

Reply:
it's the same as with the `deployment` PR